### DISCUSSION TITLE
Dudas respecto al brainstorming

### DISCUSSION THREAD

**Discussion by emrodk** [gamerevar/proyecto-comunitario-1-gd#51] [Archivados]
Premisa: la idea es que hay dos personajes que tienen que ir a un lugar por una buena razón, y al final se tienen que enfrentar y el que gana es el que consigue/cumple su buena razón. Los dos empiezan de lugares diferentes y tienen caminos diferentes, pero terminan en el mismo lugar. [sic]

La premisa indica que los juegos tienen que estar relacionados, cada juego sería una perspectiva distinta de una misma narrativa culminando ambos en un mismo final. Para que esto sea posible los dos juegos se tienen que pensar a la vez ya que un cambio en el diseño de un juego afectaría al otro.  Entonces mi duda va en si resulta conveniente que esta etapa de brainstorming suceda de manera separada entre ambos equipos

**Comment by LeandroCuello** [comment]
Opino lo mismo. Para mi, por ahora pongamos todas las ideas en un repositorio. Realmente no me importa en cual.

**Reply by emrodk** [reply]
Coincido, otra duda que me surge es si habría assets en común ya que el universo sería el mismo. Si fuera el caso, creo que habría que considerar crear un tercer repo con tal de unificar el diseño y tener un banco común de assets. Ese repo después se puede usar cómo submodulo en el repo de cada engine

**Reply by gamerevar** [reply]
Cada juego va a tener su estetica y su mecanica. Uno puede ser un juego por turnos isometrico y el otro un point and click. La estetica la va a terminar definiendo el lead de cada proyecto, por ahora solo aconsejaria que tiren ideas al respecto, pero no se fijen en que quedará asi. 

En cuanto a utilizar un solo repo para brainstorm, es un punto válido. Lo vamos a ver con los lead design y actualizaremos como seguimos. Por el momento continuen con las ideas, de una u otra forma, todas van a ser tenidas en cuenta por mas q esten en repos diferentes.

**Comment by gamerevar** [comment]
Bueno, vamos a utilizar este repositorio como el **único habilitado** para brainstorming. Finalizada la etapa, o para cualquier otro tema, cada proyecto continuará utilizando el que le corresponda.
